samples/bluetooth/central_gatt_write/src/gatt_write_common.c

```diff
@@ -12,10 +12,20 @@
 #include <zephyr/bluetooth/gatt.h>
 #include <zephyr/bluetooth/hci.h>
 
+/* Interval between storing the measured write rate */
+#define METRICS_INTERVAL 1U /* seconds */
+
 static struct bt_gatt_exchange_params mtu_exchange_params;
 static uint32_t write_count;
 static uint32_t write_len;
 static uint32_t write_rate;
+
+#if defined(CONFIG_BT_USER_PHY_UPDATE)
+/* Count down number of metrics intervals before performing a PHY update */
+#define PHY_UPDATE_COUNTDOWN 3U
+static uint32_t phy_update_countdown;
+#endif /* CONFIG_BT_USER_PHY_UPDATE */
```

Comment:
Suggest not to guard this with `CONFIG_BT_USER_PHY_UPDATE` and use IS_ENABLED(CONFIG_BT_USER_PHY_UPDATE` in the function below. The linker will remove the unused variables if `CONFIG_BT_USER_PHY_UPDATE=n` and it makes for a cleaner code

Reply:
updated